app/routes/projects/$projectId/editContributorsPath.tsx

```diff
@@ -0,0 +1,499 @@
+        >
+          <GoBack title="Back to project" href={`/projects/${projectId}`} />
+
+          <EditPanelsStyles>
+            <Box>
+              <Tabs
+                value={tabIndex}
+                onChange={handleTabChange}
+                aria-label="Edit project"
+              >
+                <Tab
+                  component={Link}
+                  label="Project Details"
+                  to={`/projects/${projectId}/edit`}
+                />
+                <Tab
+                  component={Link}
+                  label="Contributor's Path"
+                  to={`/projects/${projectId}/editContributorsPath`}
+                />
+              </Tabs>
+            </Box>
+
+            <TabPanel value={tabIndex} index={0}></TabPanel>
+            <TabPanel value={tabIndex} index={1}>
+              <Grid
+                container
+                justifyContent="flex-end"
+                alignItems="center"
+                spacing={2}
+                mb={4}
+              >
+                <Grid item>
+                  <Link to={`createStage`}>
+                    <Button variant="contained">Add Stage</Button>
+                  </Link>
+                </Grid>
+              </Grid>
+
+              {projectStages?.map((stage, index) => (
+                <Stack
+                  key={stage.id}
+                  mb={2}
+                  padding={4}
+                  style={{
+                    border: "2px solid #e2e2e2",
+                    borderRadius: "1em",
+                  }}
+                >
+                  <Grid
+                    container
+                    justifyContent="space-between"
+                    alignItems="center"
+                  >
+                    <Grid item>
+                      <h2>{stage.name}</h2>
+                    </Grid>
+                    <Grid item>
+                      <Grid
+                        container
+                        justifyContent="flex-end"
+                        alignItems="center"
+                        spacing={2}
+                      >
+                        <Grid item>
+                          <Link to={`deleteStage?id=${stage.id}`}>
+                            <Button variant="contained">Delete Stage</Button>
+                          </Link>
+                        </Grid>
+                        <Grid item>
+                          <Link to={`editStage?id=${stage.id}`}>
+                            <Button variant="contained">Edit Stage</Button>
+                          </Link>
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                  <Grid
+                    container
+                    spacing={2}
+                    mt={2}
+                    justifyContent="space-between"
+                    alignItems="stretch"
+                  >
+                    <Grid
```

Comment:
I think that this grid could be a Card element, in order to avoid the two borders, criteria and mission, I'm not sure about these borders, it could be avoided by another way? what do you think? cc @leslye-patino